src/sentry/notifications/notifications/integration_nudge.py

```diff
@@ -0,0 +1,111 @@
+    "Like getting notified in {provider}? Sentry can do that. Want to be left alone? Ignore this.",
+    "Like this notification? There's more where that came from. Hear from Sentry right here in {provider}.",
+    "Notifications aren't always bad. Hear from Sentry right here in {provider}.",
+    "Remember when email was exciting? It's not 1998 anymore. Get notified in {provider} instead.",
+    "Reminisce on the days before your email address was an obligational burden"
+    " while signing up for {provider} notifications instead.",
+    "This is a notification about notifications. Now you can hear from Sentry right here in {provider}.",
+    "This isn't 1998. Swap emails for Sentry notifications in {provider}.",
+    "We're doing something new. Get Sentry notifications right here in {provider}.",
+    "We're here to notify you about notifications. Meta, huh? Hear from Sentry right here in {provider}.",
+    "We've got something new for you. Get Sentry notifications right here in {provider}.",
+    "Who even checks their email anymore? Get Sentry notifications in {provider}.",
+    "You've got mail. So much mail. You're lost and confused as you cut through"
+    " a jungle of mail. Sentry can notify you in {provider} instead.",
+]
+
+
+class IntegrationNudgeNotification(BaseNotification):
+    category = "integration_nudge"
+    filename = "integration-nudge"
+    type = "integration.nudge"
+
+    def __init__(
+        self,
+        organization: Organization,
+        recipient: User,
+        provider: ExternalProviders,
+        seed: int | None = None,
+    ) -> None:
+        super().__init__(organization)
+        self.recipient = recipient
+        self.provider = provider
+        self.seed = (
+            (seed % len(MESSAGE_LIBRARY))
+            if seed is not None
+            else random.randint(0, len(MESSAGE_LIBRARY) - 1)
+        )
+
+    def get_reference(self) -> Model | None:
+        return None
+
+    def get_participants(self) -> Mapping[ExternalProviders, Iterable[Team | User]]:
+        return {self.provider: {self.recipient}}
+
+    def get_subject(self, context: Mapping[str, Any] | None = None) -> str:
+        return ""
+
+    def get_message_description(self) -> Any:
+        return MESSAGE_LIBRARY[self.seed].format(provider=self.provider.name.capitalize())
+
+    def get_message_actions(self, recipient: Team | User) -> Sequence[MessageAction]:
+        return [
+            MessageAction(
+                name="Turn on personal notifications",
+                action_id="enable_notifications",
+                value="all_slack",
+            )
+        ]
+
+    def get_context(self) -> MutableMapping[str, Any]:
+        return {}
+
+    def get_notification_title(self) -> str:
+        return ""
+
+    def get_title_link(self) -> str | None:
+        return None
+
+    def build_attachment_title(self) -> str:
+        return ""
+
+    def get_filename(self) -> str:
```

Comment:
@mgaeta Wondering why all these functions return an empty string. IMO it's better to let it error out because any behavior coming from this would be invalid